getnacelle/nacelle-js
# feat(shopify-checkout): checkout line items

### WHY are these changes introduced?

Addresses [LS-1658](https://nacelle.atlassian.net/browse/LS-1548)

> Implement lineItems & discountApplications fragments in mutations and queries



### WHAT is this pull request doing?

This PR brings changes from #83 & #84 into `main`.



## 

### How to Test

- Check out the `LS-1546/line-item-discounts` branch
- `cd packages/shopify-checkout`
- `npm run build` - the build process should complete without errors.
- `npm run test:ci` - all unit tests should pass.

### Testing checklist
